### PR TITLE
Add Japanese translations for #4163 and #4129

### PR DESCRIPTION
**Pull request by ykzts**
- Improve UI of admin site settings (#4163)
- Remote following success page (#4129)

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -154,24 +154,31 @@ ja:
...
+        title: カスタム詳細説明
       site_terms:
-        desc_html: プライバシーポリシーのページに表示されます。<br>HTMLタグが利用可能です。
-        title: サイトのプライバシーポリシー
-      site_title: サイトのタイトル
+        desc_html: あなたは独自のプライバシーポリシーや利用規約、そのほかの法的根拠を書くことができます。HTMLタグが使えます
+        title: カスタム利用規約
+      site_title: インスタンスの名前
+      timeline_preview:
+        desc_html: ランディングページに公開タイムラインを表示します
```
Comment (unarist) [inline config/locales/ja.yml]: Should we unify "トップページ" and "ランディングページ"?

Comment (unarist) [inline config/locales/ja.yml]: Ah, I found one more:

registrations.closed_message.desc_html: フロントページ
site_description.desc_html: トップページ
timeline_preview.desc_html: ランディングページ

Comment (ykzts) [inline config/locales/ja.yml]: `registrations.closed_message.desc_html`: frontpage
`site_description.desc_html`: frontpage
`timeline_preview.desc_html`: landing page

🤔

Comment (ykzts) [inline config/locales/ja.yml]: I followed the English version ([`en.yml` line 194](https://github.com/tootsuite/mastodon/blob/056b5ed72f6d980bceeb49eb249b8365fe8fce66/config/locales/en.yml#L194)).

**Review by unarist — APPROVED** [review]

**Review by Gargron — APPROVED** [review]

**Pull request merged**